pulsar-client-cpp/lib/ClientConnection.cc

```diff
@@ -97,6 +97,12 @@ static Result getResult(ServerError serverError) {
 
         case UnsupportedVersionError:
             return ResultUnsupportedVersionError;
+
+        case TooManyRequests:
+            return ResultTooManyLookupRequestException;
```

Comment:
ResultTooManyRequestException - in case we want to ratelimit something else in the future

Reply:
The enum value was already defined in `Result`